recipes/base_config.rb

```diff
@@ -17,6 +17,8 @@
 
 include_recipe 'aws-parallelcluster::base_install'
 
+include_recipe 'aws-parallelcluster::_nfs_config'
```

Comment:
let's make sure nfs is installed at ami build time and only configured at runtime

Reply:
Ok, including `nfs::server` in `base_install` so packages are installed at build time